cmd/sealctl/cmd/static_pod.go

```diff
@@ -40,48 +46,92 @@ func newStaticPodCmd() *cobra.Command {
 	return staticPodCmd
 }
 
+type lvscarePod struct {
+	vip    string
+	master []string
+	image  string
+	name   string
+	print  bool
+}
+
 func newLvscareCmd() *cobra.Command {
-	var vip, image, name string
-	var masters []string
-	var printBool bool
+	var obj lvscarePod
+	var setImage bool
 	var lvscareCmd = &cobra.Command{
 		Use:   "lvscare",
 		Short: "generator lvscare static pod file",
 		PreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
try to use `PreRunE` and `RunE`, let the `Execute` to handle the returned error.